cmake/FindLibClang.cmake

```diff
@@ -15,13 +15,16 @@ if( ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set(TRIQS_LIBCLANG_CXX_ADDITIONAL_FLAGS "" CACHE STRING "Additional flags to be passed to libclang when parsing with clang")
 
   set(CLANG_COMPILER "clang++")
-  set(CLANG_OPT  -std=c++1y -stdlib=libc++ )
+  set(CLANG_OPT -stdlib=libc++ )
 
 else()
 
 SET(TRIAL_LIBRARY_PATHS
+ ENV LIBRARY_PATH
```

Comment:
Ok, I see.

`ENV` is a keyword in `find_library`'s signature. Are you sure it will be recognized as such when passed via a variable? What happens if I want to search for a library in a directory called `ENV`?